docs/reference/ccr/apis/follow/get-follow-stats.asciidoc

```diff
@@ -74,6 +74,8 @@ task. In this situation, the following task must be resumed manually with the
 
 `index`::
 (string) The name of the follower index.
+`follower_to_leader_lagging_ops_count`::
```

Comment:
Naming nit: how about `total_global_checkpoint_lag`? The `follower_to_leader` bit seems redundant since this is the CCR stats API, and I think it would be helpful to include `global_checkpoint` in the name in case we decide to add some other metrics in future.

Also you're missing a blank line above this item.

Reply:
"total_global_checkpoint_lag" sounds good, thx